cwms-data-api/src/main/java/cwms/cda/api/BasinController.java

```diff
@@ -131,8 +131,8 @@ public void getAll(@NotNull Context ctx) {
     @Override
     public void getOne(@NotNull Context ctx, @NotNull String basinId) {
 
-        try (final Timer.Context timeContext = markAndTime(GET_ONE);
-             DSLContext dsl = getDslContext(ctx)) {
+        try (final Timer.Context timeContext = markAndTime(GET_ONE)) {
+            DSLContext dsl = getDslContext(ctx);
```

Comment:
are we sure the context shouldn't be in the try-with block to be closed?

Reply:
The signature of DSLContext was changed -the close() method and autocloseable interface were removed.  

https://github.com/jOOQ/jOOQ/issues/10512

Apparently we didn't need to be closing the DSLContext this entire time.

Reply:
Ah.